website/blog/2025-11-10-release-1.23.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Podman Desktop 1.23 Release
+description: Podman Desktop 1.23 has been released!
+slug: podman-desktop-release-1.23
+authors: [gastoner]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.23.png
+---
+
+import ReactPlayer from 'react-player'
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.23 Release! 🎉
+
+![podman-desktop-hero-1.23](/img/blog/podman-desktop-release-1.23/banner.png)
+
+Podman Desktop 1.23 is now available. [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements:
+
+- **Support for a managed configuration**: Loads a 'managed by' default settings file from a pre-defined location on the user's OS to support a managed configuration.
```

Comment:
```suggestion
- **Support for a managed configuration**:  IT teams can deploy pre-configured settings organization-wide. Developers get consistent, ready-to-use environments without manual setup.
```

Reply:
Managed configuration is not really a feature for every user. I would put it as the last one.